<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandwidthWidget</class>
 <widget class="QWidget" name="BandwidthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,6">
   <item>
    <layout class="QHBoxLayout" name="settingsLayout">
     <item>
      <widget class="QLabel" name="formatLabel">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Формат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="prefixComboBox">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="unitComboBox">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="chartLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
